modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -166,7 +166,7 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
         // $scan_types are set in the configuration module
         $modalities_subquery = '';
         foreach ($case_desc as $key => $value) {
-            $modalities_subquery = "$value as $scan_id_types[$key]_QC_Status";
+            $modalities_subquery .= "$value as $scan_id_types[$key]_QC_Status,";
```

Comment:
This is going to have (or I guess technically already has) similar problems to #6668 .. can you escape the field name?

Reply:
Yes, I'll push a commit shortly to fix this.

Reply:
Commit is pushed, implement DB->quote for modalities subquery.